### DISCUSSION TITLE
Improvement: Source as additional field in recipe header

### DISCUSSION THREAD

**Discussion by sg17843** [TandoorRecipes/recipes#1442] [Ideas]
Hello,

I have only been using Tandoor for a few days. In doing so, I have imported some recipes from various websites and captured others from the family recipe collection. In the case of the transfer from the web, the source is inserted at the end of the recipe as "imported from ...". And is thus m.E. not usable via the search function.

I propose to store the source information with different types like internet, cookbook, magazine, family, friends etc. as well as a graphic (logo of the website, logo of the magazine etc.) as an independent field group in the header of the recipe. This way it could also be searched for with the search function.

best regards

**Comment by smilerz** [comment]
duplicate #424
